sbin/firehol

```diff
@@ -253,26 +239,23 @@ fi
 
 # FireHOL config directory.
 # EVEN IF YOU CHANGE THIS, THE firehol-defaults.conf FILE
-# SHOULD STILL EXIST IN THE ORIGINAL $FIREHOL_CONFIG_DIR
-FIREHOL_CONFIG_DIR="$FIREHOL_CONFIG_DIR"
+# SHOULD STILL EXIST IN THE ORIGINAL $SYSCONFDIR/firehol
+FIREHOL_CONFIG_DIR="${FIREHOL_CONFIG_DIR}"
```

Comment:
all I am saying that:
```
FIREHOL_CONFIG_DIR="${FIREHOL_CONFIG_DIR}"
```
can be just removed, no? why do we need to touch a line that is useless?